tests/helpers.py

```diff
@@ -149,10 +151,10 @@ def assert_polygon_annotation_matches_dict(
         annotation_instance.annotation_id == annotation_dict["annotation_id"]
     )
     for instance_pt, dict_pt in zip(
-        annotation_instance.vertices, annotation_dict["vertices"]
+        annotation_instance.vertices, annotation_dict["geometry"]["vertices"]
```

Comment:
:nit: user constants like VERTICES_KEY here

Reply:
That goes against style everywhere else in the test, will just keep consistent for now